canvas_account_admin_tools/urls.py

```diff
@@ -10,7 +10,7 @@
     url(r'^not_authorized$', icommons_ui_views.not_authorized, name='not_authorized'),
     url(r'^tool_config$', views.tool_config, name='tool_config'),
     url(r'^lti_launch$', views.lti_launch, name='lti_launch'),
-    url(r'^icommons_rest_api/(?P<path>.*)$', views.icommons_rest_api_proxy, name='icommons_rest_api_proxy'),
     url(r'^account_dashboard$', views.dashboard_account, name='dashboard_account'),
     url(r'^course_info/', include('course_info.urls', namespace='course_info')),
+    url(r'^api/course/v2/', include('course_api.urls')),
```

Comment:
Any reason why we wouldn't update the icommons_rest_api to a more generic namespace as part of this change, since we're moving towards it as a more robust replacement for the existing one anyway? I guess it would be hard to figure out what namespace works, in a case where it would be included with other APIs in a project, but if we're moving towards running it in its own project context eventually then that's not a big issue...?